### DISCUSSION TITLE
version 2.12.0 release

### DISCUSSION THREAD

**Discussion by earth-chris** [openvinotoolkit/nncf#2818] [Q&A] [answered]
Hello,

I see that `2.12.0` has been pushed to source, but the last version that has been tagged and released is `2.11.0`. Can we expect the 2.12.0 release to be published to PyPI soon?

Cheers,

**Comment by MaximProshin** [comment, marked as answer]
Hi @earth-chris , we plan to release NNCF 2.12 on ww31

**Reply by earth-chris** [reply]
Thank you for publishing the release on time! 🙏🏻
